### PR TITLE
chore: update channels path

### PR DESCRIPTION
**Pull request by hejkerooo**

### CODE REVIEW

**Review by kim-energyweb — APPROVED** [review]
LGTM

File: apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts [hunk 1]
```diff
@@ -121,8 +127,7 @@ export class ChannelService {
     const channel = this.getChannelOrThrow(channelName);
 
     await this.channelRepository.delete(channel.fqcn);
-
-    await this.commandBus.execute(new RefreshAllChannelsCacheDataCommand());
+    await this.channelRepository.delete(channel.fqcn);
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts:130]: why delete twice?

Comment (hejkerooo) [reply apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts:130]: Somehow LokiJS despite of `saveDatabase` is not removing this record, it only works when you call it twice - which is weird. For not it is hotfix but we need to consider better solution.

Comment (vikaskumr) [reply apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts:130]: ok understood

**Review by vikaskumr — APPROVED** [review]
approved

**Review by kim-energyweb — COMMENTED** [review]
LGTM

**Pull request merged**